Makefile

```diff
@@ -22,18 +22,18 @@ deps:
 	go get -u github.com/git-chglog/git-chglog/cmd/git-chglog
 
 build:
-	go build -ldflags "-X github.com/qlcchain/go-qlc/chain.Version=${VERSION} \
-		-X github.com/qlcchain/go-qlc/chain.GitRev=${GITREV} \
-        -X github.com/qlcchain/go-qlc/chain.BuildTime=${BUILDTIME} \
-        -X github.com/qlcchain/go-qlc/chain.Mode=MainNet" -i -o $(shell pwd)/$(BUILDDIR)/$(BINARY) $(shell pwd)/$(MAIN)
+	go build -ldflags "-X github.com/qlcchain/go-qlc/chain/version.Version=${VERSION} \
+		-X github.com/qlcchain/go-qlc/chain/version.GitRev=${GITREV} \
+        -X github.com/qlcchain/go-qlc/chain/version.BuildTime=${BUILDTIME} \
+        -X github.com/qlcchain/go-qlc/chain/version.Mode=MainNet" -i -o $(shell pwd)/$(BUILDDIR)/$(BINARY) $(shell pwd)/$(MAIN)
```

Comment:
should update ldflags in `.goreleaser.yml`

Reply:
done